code/particle/effects/SingleParticleEffect.cpp

```diff
@@ -58,7 +58,9 @@ SingleParticleEffect* SingleParticleEffect::createInstance(int effectID, float m
 	Assertion(minSize >= 0.0f, "Minimum size may not be less than zero, got %f!", minSize);
 
 	auto effectPtr = new SingleParticleEffect("");
-	effectPtr->m_particleProperties.m_bitmap = effectID;
+	SCP_vector<int> bitmap_list;
+	bitmap_list.push_back(effectID);
+	effectPtr->m_particleProperties.m_bitmap_list = bitmap_list;
```

Comment:
You can also just use `push_back` on the bitmap list instead:
```suggestion
	effectPtr->m_particleProperties.m_bitmap_list.push_back(effectID);
```